src/c++/perf_analyzer/perf_utils.h

```diff
@@ -145,4 +145,49 @@ template <Distribution distribution>
 std::function<std::chrono::nanoseconds(std::mt19937&)> ScheduleDistribution(
     const double request_rate);
 
+// Calls MPI_Init API.
```

Comment:
Move to src/c++/perf_analyzer/mpi_utils.h

Reply:
Moved 👍